platforms/Windows/installer-arm64.wxs

```diff
@@ -0,0 +1,25 @@
+﻿<Wix xmlns="http://wixtoolset.org/schemas/v4/wxs" xmlns:bal="http://wixtoolset.org/schemas/v4/wxs/bal">
+  <Bundle Name="Swift Developer Package for Windows aarch64" Version="$(var.ProductVersion)" Manufacturer="swift.org" UpgradeCode="151d42d9-8877-4b72-ac62-4695243a35c1">
```

Comment:
Do we want to call it `aarch64` or `arm64`?

Reply:
I do want to rename it to ARM64, but I would rather do that in a single pass (I have exactly that change queued up :)).  This maintains the current spelling of aarch64 which is used in the MSIs.  I am going to be doing a much more invasive renaming when I do the ARM64 rename because this does get shown in a few places.